www/src/components/evaluation-table.js

```diff
@@ -116,9 +116,9 @@ class EvaluationTable extends Component {
                       <SectionHeaderBottom
                         display={row.node.Subcategory}
                         category={row.node.Subcategory}
-                        key={`section-header-bottom-${i}`}
+                        key={`section-header-${s}-bottom-${i}`}
```

Comment:
Fixing a bunch of errors in the console right? Thank you for doing this!

Reply:
👍 Yeah, quite a few :) — btw. would have loved to resolve the last remaining warning for "Features", which is the product of this https://github.com/gatsbyjs/gatsby/blob/1257a225de0bb3be8314ecc5684d1c6635d3ee05/www/src/components/evaluation-table.js#L93-L102 along the way, but gave up after 2 minutes ("illegal nesting: `tbody` in `div`" or something).

Either the scrolling lib mentioned there is still not playing nicely with anchors in tables (@calcsam that was you who wrote that comment, right?), or something with the sidebar "scrollspy" doesn't work correctly. Didn't give moving the `id` around actually, maybe that solves it — I think it's currently attached to a `td`…